experiments/tf_trainer/common/convert_csv_to_tfrecord.py

```diff
@@ -0,0 +1,41 @@
+"""A function to convert csvs to TFRecords."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import pandas as pd
+import tensorflow as tf
+
+COLS = ['comment_text', 'frac_neg']
+
+FLAGS = tf.app.flags.FLAGS
+
+tf.app.flags.DEFINE_string("input_csv_path",
+                           None,
+                           "Path to the input csv file.")
+tf.app.flags.DEFINE_string("output_tfrecord_path", None,
+                           "Path where the output TFRecord should be written.")
+
+
+def convert_csv_to_tfrecord(input_csv_path, output_tfrecord_path):
+    df = pd.read_csv(tf.gfile.Open(input_csv_path))
+    csv = df[COLS].values
+    with tf.python_io.TFRecordWriter(output_tfrecord_path) as writer:
+        for row in csv:
+            features, label = row[0], row[1]
```

Comment:
features -> text maybe, since we assume it is text when we encode

Reply:
Done.